### DISCUSSION TITLE
Multiple Object Tracking and Segmentation Challenges

### DISCUSSION THREAD

**Discussion by pikaandhouse** [bdd100k/bdd100k#248] [Q&A]
In the Multiple Object Tracking and Segmentation Challenges,my submmsion stuck in the status of running for a long time and get no response.

**Comment by CarlHuangNuc** [comment]
just wait 10 minutes

**Comment by siyuanliii** [comment]
Could you give us the challenge id? Are you submitting on our CVPR challenges?

**Comment by pikaandhouse** [comment]
Submission id :&nbsp;&nbsp;&nbsp;204640
Submitted at:&nbsp;&nbsp;&nbsp;May 31, 2022, 8:55:05 PM





------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "bdd100k/bdd100k"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年6月1日(星期三) 晚上9:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [bdd100k/bdd100k] Multiple Object Tracking and Segmentation Challenges (Discussion #248)





 
Could you give us the challenge id? Are you submitting on our CVPR challenges?

**Reply by siyuanliii** [reply]
I mean your challenge id. Are you working on challenge 1672?

**Comment by pikaandhouse** [comment]
yes&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年6月1日(星期三) 晚上9:13
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [bdd100k/bdd100k] Multiple Object Tracking and Segmentation Challenges (Discussion #248)





 
I mean your challenge id. Are you working on challenge 1672?

**Reply by siyuanliii** [reply]
I just tested this challenge evaluation. It works fine on my side. Also, I see there are successful submissions on the same date when you submitted your results. Could you please try to 1. check your submission format (whether it is correct) and 2. resubmit your results to see if problems remain.
